### PR TITLE
JBPM-6770 - Information not available in TaskLifeCycleEventListener a…

### PR DESCRIPTION
**Pull request by mswiderski**
…nd ProcessEventListener

depends on https://github.com/kiegroup/drools/pull/1712

### CODE REVIEW

**Comment by mswiderski** [comment]
Jenkins retest this

File: jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/events/TaskEventImpl.java [hunk 1]
```diff
@@ -50,5 +53,10 @@ public TaskContext getTaskContext() {
 	public void setTaskContext(TaskContext context) {
 		this.taskContext = context;
 	}
+
+    @Override
+    public Date getEventDate() {
+        return this.eventDate;
+    }
```
Comment (MarianMacik) [inline jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/events/TaskEventImpl.java]: Maybe just nitpicking, but this looks like a bad formatting.

Comment (mswiderski) [reply jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/events/TaskEventImpl.java]: reformatted

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by MarianMacik** [comment]
Hmm, it seems that this PR breaks the CDI tests because there is an AssertJ used and arquillian-junit-container  does not contain this dependency. But this PR didn't introduce the test `org.jbpm.services.cdi.test.UserTaskServiceCDIImplTest.testCompleteAutoProgressWrongDeploymentId`, it was added here https://github.com/kiegroup/jbpm/pull/1114/files#diff-6727fce3f2468f67b274acd90b4d11fcR909

#1114 has passed though.

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**